.github/workflows/nightly-benchmark-cleanup.yml

```diff
@@ -0,0 +1,168 @@
+name: Nightly Benchmark Cleanup
+
+on:
+  workflow_dispatch:  # Allow manual triggering
+    inputs:
+      benchType:
+        description: 'Type of benchmark to clean up'
+        required: true
+        type: choice
+        options:
+          - tpch
+          - auctionmark
+          - readings
+      status:
+        description: 'Status of the benchmark'
+        required: true
+        type: choice
+        options:
+          - success
+          - failure
+      nodeId:
```

Comment:
Realize nodeId is passed along with the callback and we use it to identify which pod to scrape - what sets XTDB_NODE_ID?

Reply:
It's set in the benchmark-x.yaml

          env:
            - name: XTDB_NODE_ID
              valueFrom:
                fieldRef:
                  fieldPath: metadata.name